examples/python/Makefile

```diff
@@ -32,33 +32,60 @@
 # environment.mk verifies the build environment and sets the following
 # makefile variables:
 #
-# TESTBENCH_PATH: The path to the testbench directory in the bsg_f1 repository
-# LIBRAIRES_PATH: The path to the libraries directory in the bsg_f1 repository
-# HARDARE_PATH: The path to the hardware directory in the bsg_f1 repository
+# LIBRAIRES_PATH: The path to the libraries directory
+# HARDWARE_PATH: The path to the hardware directory
+# EXAMPLES_PATH: The path to the examples directory
```

Comment:
This file is an amalgamation/move of the files tests.mk, and rules.mk and Makefile in regression/python/